EULA.txt

```diff
@@ -0,0 +1,79 @@
+MICROSOFT SOFTWARE LICENSE TERMS
```

Comment:
Can we move this into e.g. `sdk/communiation/AzureCommunicationCalling/` ? I think seeing this in the root of the repo might concern users. I'm fine if this EULA is the only file in that directory.

Reply:
Moved